cmd/swarmctl/service/update.go

```diff
@@ -36,6 +36,16 @@ var (
 				return err
 			}
 
+			// change the image if that positional arg is present
+			if len(args) > 1 {
+				spec.Task.GetContainer().Image = args[1]
+			}
+
+			// change the args too, if those positional args are present
```

Comment:
`args` is difficult to handle gracefully because you can't create a multi-valued flag with the flag library we use. The only halfway decent solution I could think of is enclosing the arg in quotes and then parsing it into an array of arguments with a big script (respecting things like backticks and quotes)